0001-Support-musllinux-armv6l.patch

```diff
@@ -1,27 +1,36 @@
-From 1c9bba7af44d2465d3657f7a63f8a0ffdca3239b Mon Sep 17 00:00:00 2001
+From 523ad69294a1190812e63fb78ff94c8f5359011f Mon Sep 17 00:00:00 2001
 From: Pascal Vizeli <pvizeli@syshack.ch>
 Date: Wed, 15 Jun 2022 12:48:56 +0000
 Subject: [PATCH] Support musllinux armv6l
 
----
- src/auditwheel/policy/__init__.py           | 1 +
- src/auditwheel/policy/musllinux-policy.json | 4 ++++
- 2 files changed, 5 insertions(+)
+Co-authored-by: Marc Mueller <30130371+cdce8p@users.noreply.github.com>
 
+diff --git a/src/auditwheel/architecture.py b/src/auditwheel/architecture.py
+index 4a506e8..f0c39d6 100644
+--- a/src/auditwheel/architecture.py
++++ b/src/auditwheel/architecture.py
+@@ -11,6 +11,7 @@ class Architecture(Enum):
+     value: str
+ 
+     aarch64 = "aarch64"
++    armv6l = "armv6l"
+     armv7l = "armv7l"
+     i686 = "i686"
+     loongarch64 = "loongarch64"
 diff --git a/src/auditwheel/policy/__init__.py b/src/auditwheel/policy/__init__.py
-index f38ab43..8ed9db8 100644
+index 29a743c..36c9ba2 100644
 --- a/src/auditwheel/policy/__init__.py
 +++ b/src/auditwheel/policy/__init__.py
-@@ -285,6 +285,7 @@ def _fixup_musl_libc_soname(libc: Libc, arch: str, whitelist):
-             "aarch64": "libc.musl-aarch64.so.1",
-             "s390x": "libc.musl-s390x.so.1",
-             "ppc64le": "libc.musl-ppc64le.so.1",
-+            "armv6l": "libc.musl-armvhf.so.1",
-             "armv7l": "libc.musl-armv7.so.1",
-             "riscv64": "libc.musl-riscv64.so.1",
-         }
+@@ -295,6 +295,7 @@ def _fixup_musl_libc_soname(
+             Architecture.aarch64: "libc.musl-aarch64.so.1",
+             Architecture.s390x: "libc.musl-s390x.so.1",
+             Architecture.ppc64le: "libc.musl-ppc64le.so.1",
++            Architecture.armv6l: "libc.musl-armhf.so.1",
```

Comment:
Note the libc filename is actually `libc.musl-armhf.so.1` (without the `v`). See https://pkgs.alpinelinux.org/contents?name=musl&repo=main&branch=v3.22&arch=armhf

Reply:
Uh I see, nice find! Shouldn't this have caused issues for armv6l builds? :thinking:

Reply:
Ah I see, nice!